index.js

```diff
@@ -0,0 +1,93 @@
+const startButton = document.querySelector(".start-button");
```

Comment:
```suggestion
const startButton = document.querySelector('.start-button');
```
파일 내에서 "(큰따옴표)나 '(작은따옴표) 중 하나로 통일해서 사용해주세요!
템플릿리터럴을 위해 사용하는 `(백틱)은 함께 사용하시면 됩니다.